### DISCUSSION TITLE
Thoughts on the free merchant power structure

### DISCUSSION THREAD

**Discussion by I-am-Erk** [CleverRaven/Cataclysm-DDA#52150] [General]
It's getting close to time to finish off the free merchants by building their bunker and establishing the rest of their personnel. The bulk of the free merchants are caravaners and guards, who are "repaid" for the risks they take traveling the world by getting a safe place to hole up underground, access to food and medical care, etc. There are some people who just live underground, though, and never leave. These people need some reason to be allowed to stay down there in safety. I think they fall into three categories: Top leadership, children, and a few dedicated childcare people. 

A first pass on mapping would build a central common area of the bunker with some office space. There would be suggestions of storage space and additional areas that are guarded and inaccessible, and not actually mapped. We can now add children by using the "monsters that can talk" framework which is a realistic depiction of children for anyone who has ever tried to raise them. We should consider if we want children to function as regular monsters in game or if we want them to just vanish at the first sign of conflict to avoid problems with ratings in other countries.

## Leadership
I think we should have a few top-level free merchants in a sort of oligarchy. They'll need names so we can refer to them in conversation more.
-"Terry", broker: the only important person seen regularly on the surface. Manages trade agreements and things, liases with the caravans, etc. Kind of the boss of the people running the front merchant desk and the caravaneers combined.
-"Chris", quartermaster: the person who appointed themselves in charge of managing all the supplies and doling them out, early on, and turned out to be fairly good at it so stayed in that position. Works closely with the broker, but they're about on equal footing. Never leaves the basement. Original job should be just shy of important enough to go to a private bunker.
-"Sam", scheduler: responsible for working out who does what duties fairly. Was originally the person responsible for sorting out who came into the base and who didn't, so for a while was the most important person. Was downgraded to this job, but remains quite important. Total and utter bastard, but has a following among the merchants due to being "someone who took charge" and also due to undercurrent of racism that agrees with them.
-"Charlie", maintainer: a retired shop teacher(?) responsible for keeping the base itself running. Super nice, hates the quartermaster and scheduler with a passion, which gives them lots of problems politically, but there is nobody else who can do their job.

Those four are pretty much equal in importance. Major decisions are put to a vote but the topics of vote tend to be decided by those four, in the way of meetings - most of the time they're doing all the talking.  I think the surface refugees don't actually get a vote currently and that is something you could potentially fight for. They may not even realize there are things being put to a vote that they're left out on.

That gives us a main dividing line: broker/maintainer who are people with practical skills and generally uninterested in the politics, and quartermaster/scheduler who (feel that) they have no actual qualifications and so have to make themselves indispensable so nobody catches on, which they do via political machinations and loyalty generation. Sam would be the clear "villain" here, but Chris is not a ton better. Before the faction even formed, they (sam especially) made sure that they would have a few likeminded people on the inside to support them, hence the frustration with how people were let into the center. Now they hold controls over most of the resource allocation - who gets the good jobs, who gets the good food - which in turn means they can cement loyalty very firmly.

There's a second division. Terry is the only leader brave enough to spend the days above ground. Even Charlie is unwilling to leave the bunker unless necessary. Terry is also the person with the most soft spot for the surface refugees, although notably, the surface refugees are still on the surface and being left out of votes, so Terry isn't exactly a bleeding heart for them either.

**Comment by PatrikLundell** [comment]
I'd make about half of them men and half of them women. Also, the "Sam" person seems to match fairly well with a prejudiced person someone on the surface essentially described as deliberately losing the paperwork for, so I'd try to match that person to that description.

It certainly make sense to have arse licking bastards toadying to an asshole leader while kicking at everyone below them, and have people more interested in their own luxury and satisfaction supported by power play as that asshole leader. Divide-and-conquer is a commonly used technique by despots to ensure their underlings oppose each other and so never get the chance to stage a coup (which typically fails anyway, eventually, but it can work for decades [the dictators of Libya and Yugoslavia are examples, although this is obviously a much smaller scale]).
